next-packages/core/__tests__/css.js

```diff
@@ -24,62 +23,13 @@ test('thing', () => {
   expect(tree.toJSON()).toMatchSnapshot()
 })
 
-test('keyframes', () => {
-  const animation = keyframes(css`
-    from {
-      color: green;
-    }
-    to {
-      color: hotpink;
-    }
-  `)
-  const tree = renderer.create(
-    <div>
-      <div
-        css={css`
-          animation: ${animation.name} ${true};
-          ${animation.styles};
-        `}
-      >
-        {animation.name}
-      </div>
-    </div>
-  )
-
-  expect(tree.toJSON()).toMatchSnapshot()
-})
-
-test('keyframes insert in css call', () => {
-  const animation = keyframes(css`
-    from {
-      color: green;
-    }
-    to {
-      color: hotpink;
-    }
-  `)
-  const tree = renderer.create(
-    <div>
-      <div
-        css={css`
-          animation: ${animation.name};
-          ${animation.styles};
-        `}
-      >
-        {animation.name}
-      </div>
-    </div>
-  )
-
-  expect(tree.toJSON()).toMatchSnapshot()
-})
-
 test('css call composition', () => {
-  const first = css`
+  let first = css`
```

Comment:
why the const -> let change?

Reply:
No real reason, I just felt like it. https://jamie.build/const.html